README.md

```diff
@@ -1,3 +1,191 @@
-# React-Bookstore
+![](https://img.shields.io/badge/Microverse-blueviolet)
 
-creating a book store app
\ No newline at end of file
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+<!--   - [🚀 Live Demo](#live-demo) -->
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+<!--   - [Run tests](#run-tests) -->
+<!--   - [Deployment](#deployment) -->
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+<!-- - [❓ FAQ (OPTIONAL)](#faq) -->
+- [📝 License](#license)
+
+
+# 📖 [Bookstore App] <a name="about-project"></a>
+
+**[Book-store app]** - This is a website that shows mastery in React and Redux, JavaScript, HTML, CSS.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://reactjs.org/">React.js</a></li>
+    <li>`Linters` `React` `GitHub` `Git` `Jest` `Webpack` `HTML` `CSS`<li/>
+  </ul>
+</details>
+
+### Key Features <a name="key-features"></a>
+
+- **[React & Redux/Javascript]**
+- **[HTML]**
+- **[Router]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+<!-- ## 🚀 Live Demo <a name="live-demo"></a>
+
+> Add a link to your deployed project.
+
+- [Live Demo Link](https://google.com)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p> -->
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+- Copy [link] (https://github.com/unorjikingsley/react-bookstore.git)
+
+### Setup
+
+Clone this repository to your desired folder:
+
+- Open terminal or command prompt and run git clone https://github.com/unorjikingsley/react-bookstore.git
+
+### Install
+
+Install this project with:
+
+- cd react-bookstore
+- run `npm i`
+
+### Usage
+
+To run the project, execute the following command:
+
+- Run `npm run start`
+
+<!-- ### Run tests
+
+To run tests, run the following command: -->
+
+<!--
+Example command:
+
+```sh
+  bin/rails test test/models/article_test.rb
+```
+--->
+
+<!-- ### Deployment
+
+You can deploy this project using: -->
+
+<!--
+Example:
+
+```sh
+
+```
+ -->
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Unorji Kingsley**
+
+- GitHub: [@githubhandle](https://github.com/unorjikingsley)
+- LinkedIn: [LinkedIn](www.linkedin.com/in/unorjikingsley)

```

Comment:
![image](https://github.com/unorjikingsley/React-Bookstore/assets/93039359/af6ca95c-f0d9-42d2-9181-a8b135a17ac7)

- [ ] Kindly fix your LinkedIn by just adding `https://` at the start of the link because right now it is broken 😨